shared/common-adapters/user-proofs.native.js

```diff
@@ -51,7 +51,7 @@ function ProofRow ({proof, onClickStatus, onClickProfile, hasMenu, style}: Proof
 
   return (
     <Box style={{...styleRow, ...style}} key={`${proof.id}${proof.type}`}>
-      <Box style={iconContainer}>
+      <Box style={{...iconContainer}}>
```

Comment:
ugh, I was testing something with different styles and was adding things to it. I forgot to not commit that change